tests/dialects/test_tsql.py

```diff
@@ -1058,18 +1058,18 @@ def test_date_diff(self):
             },
         )
         self.validate_all(
-            "SELECT DATEDIFF(year, '2020/01/01', '2021/01/01')",
+            "SELECT DATEDIFF(year, '2020-01-01', '2021-01-01')",
```

Comment:
I changed this test because Spark doesn't recognize the `yy/mm/dd` format and so it returns `NULL`, which means the transpilation is not correct.